docs/Catalog/RemovingStacks.md

```diff
@@ -0,0 +1,61 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+From time to time, you may need to remove a Quilt stack from your AWS account.
+To do this, you will need to delete the CloudFormation template that created the
+stack, which will remove all resources associated with the stack.
+
+### NOTE: Pausing the Stack Instead of Deletion
+
+If you prefer to suspend the Quilt stack temporarily, consider pausing services
+(e.g., shutting down ECS) to reduce costs. Note, however, that some charges may
+continue even with ECS inactive. To fully stop charges, you must delete the
+CloudFormation stack.
+
+## Step 1: Confirm Stack Deletion Readiness
+
+Deleting the CloudFormation stack will permanently remove any stack-specific
+configurations, analytics data, audit logs, and Athena querying setups. Data in
+your own S3 buckets will remain intact, as they are outside of CloudFormation’s
+control.
+
+### Important Considerations
+
+1. **Backup of Analytics and Audit Logs**: If you need to retain any analytics
+   or audit logs stored in stack-managed buckets, back up these files. Note that
+   **you must delete these contents after backup** to allow CloudFormation to
+   delete the bucket.
+2. **Athena Configurations**: Athena databases and Glue resources associated
+   with stack analytics data will be removed. To restore querying capabilities
+   later, you may need to reconfigure these settings.
+
+## Step 2: Attempt Stack Deletion in CloudFormation
+
+1. In the **CloudFormation** console for the appropriate AWS region, locate your
+   Quilt stack: Cloud Formation > Stacks -> Your Stack Name.
+2. Select **Delete** to begin the stack deletion process. AWS will attempt to
+   delete all resources managed by the stack.
+
+### Non-Empty S3 Buckets
+
+CloudFormation cannot delete non-empty S3 buckets. If any buckets created by the
+stack contain data, the initial deletion attempt will fail. You have two
+options:
+
```

Comment:
We should describe any buckets created by the stack that will need to be manually emptied. I know of two:
1. AnalyticsBucket
2. CloudTrailBucket

Reply:
I think we should not list these buckets because this list will be outdated quite soon and you can just check a list of resources with `DELETE_FAILED` status
it doesn't make much sense to empty buckets before trying to delete stack because running stack will fill them with new data

Reply:
Going with Sergey's recommendation.